src/Reflector/Reflector.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace BetterReflection\Reflector;
+
+interface Reflector
+{
+    public function reflect($symbolName);
```

Comment:
No, a reflector takes a string as the only parameter, the "Generic" reflector is not a reflector, but takes a `Symbol` as it's parameter